### PR TITLE
add Dominion

### PR DESCRIPTION
**Pull request by sdlax9**
updated Sources
updated game names
updated equipment types
updated sample utterances to include ‘we’
added dominion json file

### CODE REVIEW

**Comment by sdlax9** [comment]
I'm new to this...I hope I added everything correctly

File: alexa/sample_utterances.txt [hunk 1]
```diff
@@ -17,6 +17,22 @@ StartingEquipmentInGame how many {Type} you start with in {Game}
 StartingEquipmentInGame how much {Type} you start with in {Game}
 StartingEquipmentInGame how many {Type} I start with in {Game}
 StartingEquipmentInGame how much {Type} I start with in {Game}
+StartingEquipmentInGame how many {Type} do we get in {Game}
+StartingEquipmentInGame how much {Type} do we get in {Game}
+StartingEquipmentInGame how many {Type} do we get in {Game}
+StartingEquipmentInGame how much {Type} do we get in {Game}
```
Comment (btelle) [inline alexa/sample_utterances.txt]: This file shouldn't have duplicate entries, one per question phrasing works

**Review by btelle — CHANGES_REQUESTED** [review]
Overall this looks good, left a comment on sample utterances. Fix that and it should be good to go

**Comment by btelle** [comment]
Hey, I merged this in. I just added some test cases for it and fixed a couple of syntax problems in the file in the master branch. I'll add some docs on creating tests if you're interested in contributing more. Thanks for the PR!

**Comment by sdlax9** [comment]
Sweet sounds good. I'll definitely keep at it, thanks for putting this up! 

I just saw your post on hackster too, so I'll try to do some testing on my own before the next PR. As I said I'm just getting started using github, AWS, and the skills kit so I really appreciate you putting up the documentation and such, it's been a great guide.

**Pull request merged**